configs/config.yaml

```
@@ -0,0 +1,34 @@
+train:
+  batch_size: 64
+  epochs: 5
+  log_interval: 100
+  generator_lr: 1e-4
+  discriminator_lr: 1e-5
+  num_workers: 4
+  # for continuing training
+  load_checkpoint: # change from None to ckeckpoint path
+  more_epochs: 20 # How many more epochs to train
+
+logger:
+  checkpoint_path: "checkpoints/"
+  chart_path: "tmp/"
+
+system:
+  device: "cuda:0"
+  seed: 23
+
+discriminator:
+  input_channels: 1
+  input_size: 250
+  neurons: [64, 128, 256, 512]
+  beta: 0.5
+
+generator:
+  nz: 50
```

```suggestion
  n____z____: 50
```

Can you give this an unabbreviated name? :)

---

Sure, nz is number of z dimensions (z being the latent vector, or simply the noise). Would z_dimensions?

---

Hmm, I think this should rather indicate that it's the input size for the generator 🤔

---

I think that nz is sort of a convention for DCGAN, but we can break it and set it to a more descriptive `noise_size`?

---

`noise_size` is fine 👍